Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_5.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+##### CrowdStrike Falcon - SIEM ingestion Get Incident Data
+- Playbook improvements.
```

Comment:
Please add a more detailed description

Reply:
Will add more context once the build is ok